package.json

```diff
@@ -13,10 +13,12 @@
     "babel": "^5.8.20",
     "css-modules-loader-core": "0.0.12",
     "in-publish": "^2.0.0",
+    "isparta": "^3.0.3",
     "mocha": "^2.2.5"
   },
   "scripts": {
-    "test": "npm run -s build && mocha --compilers js:babel/register",
+    "test": "mocha --compilers js:babel/register",
```

Comment:
No need to make build before tests anymore. Tests are running fine on original ES6 source code, since mocha uses babel compiler.